### DISCUSSION TITLE
Module not found: Error: Can't resolve 'graphql/validation/rules/KnownFragmentNames'

### DISCUSSION THREAD

**Discussion by gautierrollin** [graphql/graphiql#1518] [General] [answered]
Hi,

I've been getting these errors with `graphiql` v0.17.5 since the upgrade of `graphql` to v15.0.0:

```
ERROR in ./node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
Module not found: Error: Can't resolve 'graphql/validation/rules/KnownFragmentNames'
```
```
ERROR in ./node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
Module not found: Error: Can't resolve 'graphql/validation/rules/NoUnusedFragments'
```

I know that you are currently rewritting GraphiQL for the v1.0.0 and that this bug is already fixed in the master branch but will you publish a fix before?

Thank's!

**Comment by acao** [comment, marked as answer]
hey there!

this was already addressed in a pre-release, you can check the changelogs to see which version exactly, but one of the first 1.0.0 alphas (more will be added).

the quick way to install the latest pre-release is `yarn add graphiql@next` or `npm install --save graphiql@next`

if you have any other questions, please feel free to reach out on discord!

**Comment by hemel-cse** [comment]
Same error with nextjs

**Comment by acao** [comment]
@hemel-cse are you using `GraphiQL` component in nextjs?

**Comment by duahaudo** [comment]
Still happen with clean `create-react-app`
```
./node_modules/graphql-language-service-utils/esm/validateWithCustomRules.js
Module not found: Can't resolve 'graphql/validation/rules/KnownFragmentNames' in 'xxxxxx/node_modules/graphql-language-service-utils/esm'
```

```
"dependencies": {
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.3.2",
    "@testing-library/user-event": "^7.1.2",
    "graphiql": "^0.17.5",
    "graphql": "^15.0.0",
    "isomorphic-fetch": "^2.2.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "3.4.1"
  }
```
is there any workaround to make it works ?

**Comment by acao** [comment]
@duahaudo please use the pre-release! `yarn add graphiql@next`
